### PR TITLE
Use bash functions

### PR DESCRIPTION
**Pull request by ThePigPeppa**

### CODE REVIEW

**Comment by ThePigPeppa** [comment]
@angristan

File: wireguard-install.sh [hunk 1]
```diff
@@ -80,12 +100,21 @@ else
...
 fi
 
+function wireguard-install() {
 # Install WireGuard tools and module
 if [[ "$OS" = 'ubuntu' ]]; then
     add-apt-repository ppa:wireguard/wireguard
     apt-get update
     apt-get install "linux-headers-$(uname -r)"
     apt-get install wireguard iptables
+elif [[ "$OS" = 'raspbian' ]]; then
```
Comment (angristan) [inline wireguard-install.sh]: Please make a separate PR for that, thanks!

Comment (ThePigPeppa) [reply wireguard-install.sh]: master, we can't just make a separate PR for that only we would also have to move the distro code too.
